client/engine/store/mockstore.go

```diff
@@ -129,7 +129,12 @@ func (ms *MockStore) SetObjective(obj protocols.Objective) error {
 			if err != nil {
 				return fmt.Errorf("error setting channel %s from objective %s: %w", ch.Id, obj.Id(), err)
 			}
-
+		case *consensus_channel.ConsensusChannel:
```

Comment:
I didn't add a test for this as it will be tested once this [TODO](https://github.com/statechannels/go-nitro/blob/ag/vf-integration/client/engine/store/mockstore_test.go#L46:L46) is resolved